language/move-prover/interpreter-testsuite/tests/concrete_check/vector.move

```diff
@@ -0,0 +1,35 @@
+module 0x2::A {
+    use 0x1::Vector;
+
+    struct Foo has drop {}
+    struct Bar {}
+
+    fun test_natives<T>(x1: T, x2: T): (T, T) {
```

Comment:
This whole file is copied from the functional-tests suite.

It might be OK to duplicate the code here in the short run but we could start to think about porting some of the functional-tests into the unit testing framework. @tzakian